src/main/java/Duke.java

```diff
@@ -1,10 +1,42 @@
+import java.util.Scanner;
+
 public class Duke {
+    private final static String LINE_BREAK = " ____________________________________________________________";
+    private final static String GREETING = "Hello! I'm Duke\n  What can I do for you?";
+    private final static String EXITING = "Bye. Hope to see you again soon!";
+    private final static Tasks tasks = new Tasks();
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        Scanner scanner = new Scanner(System.in);
+        speak(GREETING);
+        boolean running = true;
+        while (running) {
+            String command = scanner.nextLine();
+            String[] commandByWord = command.split(" ");
+            if (command.equals("bye")) { // whats the difference between equals and ==
+                speak(EXITING);
+                running = false;
+            } else if (command.equals("list")) {
+                speak(tasks.listTasks());
+            } else if (commandByWord[0].equals("mark")) {
+                speak(tasks.mark(Integer.parseInt(commandByWord[1])));
```

Comment:
Avoid the use of complicated expression and try to split up the code into intermediate steps.
```suggestion
                Integer taskIndex = Integer.parseInt(commandByWord[1]);
                String messageToUser = tasks.mark(taskIndex);
                speak(messageToUser);
```